packages/kbn-alerts-ui-shared/src/maintenance_window_callout/index.tsx

```diff
@@ -28,8 +28,10 @@ const MAINTENANCE_WINDOW_RUNNING_DESCRIPTION = i18n.translate(
 
 export function MaintenanceWindowCallout({
   kibanaServices,
+  categories,
 }: {
   kibanaServices: KibanaServices;
+  categories?: string[];
```

Comment:
I was wondering if we should do that in the API too. Maybe we can create a tech debt ticket to do it right in a long term